Reporting/M4 Document/Functionality Updates.md

```diff
@@ -0,0 +1,18 @@
+### <a name="_f7qlqezbzr2o"></a>Functionality Updates
+- So far in our project, we have implemented many features together with their tests:
+  - Registration functionality: 
+    - The constraint function (ensuring the users input their information in the correct format) is working.
+    - The register function is also working by posting the correct registered information to the database. Both of these functions pass the ad-hoc, and unit tests that we have implemented so far.
+  - Login functionality:
+    - The constraint function (ensuring the users input their information in the correct format) is working.
```

Comment:
Indeed. Not only that, but we also have client-side validation so we can guide users towards the expected format.